src/features/bridge/context/helpers/getBridgeFee.ts

```diff
@@ -0,0 +1,55 @@
+import { parseUnits } from 'ethers/lib/utils';
+import { getBridgeFeeData } from '@lib';
+import { Config, Token } from '@lib/bridgeSDK/models/types';
+
+interface GetFeeDataModel {
+  amountTokens: string;
+  isMaxOptions: boolean;
+  selectedTokenFrom: Token;
+  selectedTokenDestination: Token;
+  setTemplateDataLoader: (value: boolean) => void;
+  bridgeConfig: Config | null;
+}
+export const getFeeData = async ({
+  bridgeConfig,
+  amountTokens,
+  selectedTokenFrom,
+  selectedTokenDestination,
+  isMaxOptions
+}: GetFeeDataModel) => {
+  // we need to subtract 1 from the balance to avoid inaccuracies in calculations about the maximum transfer amount
+  const amountToSubtract = '1';
+  const balanceToFee = selectedTokenFrom.isNativeCoin
+    ? // @ts-ignore
+      selectedTokenFrom.balance.sub(
+        parseUnits(amountToSubtract, selectedTokenFrom.decimals)
+      )
+    : selectedTokenFrom.balance;
+
+  const isAmountToFeeLessThenZero = !balanceToFee.gt(0);
```

Comment:
balanceToFee.isZero()

Reply:
balanceToFee.isZero() return true only for 0. !balanceToFee.gt(0) return true if amount less then 0